### DISCUSSION TITLE
update() in server side

### DISCUSSION THREAD

**Discussion by AhmedAlsudairy** [nextauthjs/next-auth#9862] [Help]
How I can update session in server side using authV5 in nextjs latest update

I tried :

import { update } from "@/auth

But the update not in auth

**Comment by Bernardomennndes** [comment]
Actually, the NextAuth instance return a method called `unstable_update` which you could reassign the value to `update` for readability issues. Then, you just need to invoke the method on a RSC, using the "user server" tag if you want to pass to a child client component and make it interactable.

https://authjs.dev/reference/next-auth#unstable_update
